templates/public-profile.php

```diff
@@ -157,9 +157,22 @@
 				</div>
 
 				<div class="tutor-social-container content-for-mobile">
-					<?php echo wp_kses( $social_media, $allowed_html_for_social ); ?>
```

Comment:
Instead of removeing this line, just add `href` to `$allowed_html_for_social`

Reply:
updated